packages/typespec-ts/test/modularUnit/scenarios/samples/parameters/parameterTypesCheck.md

```diff
@@ -64,7 +65,8 @@ op read(@bodyRoot body: Widget): { @body body: {}};
       "plainTimeProp": "13:06:12",
       "utcDateTimeProp": "2022-08-26T18:38:00Z",
       "offsetDateTimeProp": "2022-08-26T18:38:00Z",
-      "durationProp": "P123DT22H14M12.011S"
+      "durationProp": "P123DT22H14M12.011S",
+      "withEscapeChars": "\"Tag 10\".Value"
```

Comment:
@kazrael2119 could you add more escape cases here? I believe we met some in swagger gen.